src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,68 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "movie_session")
```

Comment:
according to the naming convention table name should be plural

```suggestion
@Table(name = "movie_sessions")
```